packages/terra-folder-tree/src/subcomponents/FolderTreeItem.module.scss

```diff
@@ -29,7 +29,13 @@
     padding: 0;
   }
 
-  .radio:hover {
-    cursor: pointer;
+  .radio {
+    height: 14px;
+    margin: 17px;
+    width: 14px;
```

Comment:
Shouldn't these values have theme variables?

Reply:
I checked with @eawww, these values are the same for all themes

Reply:
Will they be the same for the Redwood theme and its 3 data density variants?

Reply:
@eawww Do you have any ideas?

Reply:
I'm not sure. @jeremyfuksa 👆?

Reply:
I won't hold up the review for this.  Just asking as it seem like something that could change.  Honestly, I am surprised the value isn't different between the default theme and the fusion theme.  It might just be another JIRA to log.